betterlockscreen

```diff
@@ -140,6 +123,25 @@ failsafe() {
         --noinput-text="" --force-clock
 }
 
+store_dunst_paused_state="$(dunstctl is-paused)"
+
+# called before screen is locked
+prelock() {
```

Comment:
Can you please move preLock back before lock for a less big change/rebasing of other PRs waiting